tests/SpecTests/ResultExpectation.php

```diff
@@ -46,8 +46,7 @@ final class ResultExpectation
     private $assertionCallable;
 
     /**
-     * @param integer $assertionType
-     * @param mixed   $expectedValue
+     * @param mixed $expectedValue
```

Comment:
Not relevant. If you'd like to just search for `@param mixed` and delete most of those I won't object (no pun intended).

Reply:
Removed this and a number of other superfluous type annotations.